### ISSUE TITLE
Many analyzer warnings in angular2_material

### ISSUE THREAD

**Issue by kevmoo** [angular/angular#1395]
Make this change to `tools/build/dartanalyzer.js`

```diff
     function analyze(dirName, done) {
       // analyze files in lib directly – or you mess up package: urls
-      var sources = [].slice.call(glob.sync('lib/*.dart', {
+      var sources = [].slice.call(glob.sync('lib/**/*.dart', {
         cwd: dirName
       }));
```

and you get...

```
dist/dart/angular2_material:[warning] The getter 'disabled_' is not defined for the class 'MdCheckbox' (/angular2_material/lib/src/components/checkbox/checkbox.dart, line 39, col 17)
dist/dart/angular2_material:[warning] The setter 'disabled_' is not defined for the class 'MdCheckbox' (/angular2_material/lib/src/components/checkbox/checkbox.dart, line 42, col 10)
dist/dart/angular2_material:[warning] The getter 'disabled_' is not defined for the class 'MdCheckbox' (/angular2_material/lib/src/components/checkbox/checkbox.dart, line 43, col 34)
dist/dart/angular2_material:[warning] Undefined class 'KeyboardEvent' (/angular2_material/lib/src/components/checkbox/checkbox.dart, line 45, col 13)
dist/dart/angular2_material:[warning] Undefined name 'KeyCodes' (/angular2_material/lib/src/components/checkbox/checkbox.dart, line 46, col 26)
dist/dart/angular2_material:[hint] Unused import (/angular2_material/lib/src/components/checkbox/checkbox.dart, line 6, col 8)
dist/dart/angular2_material:[warning] Undefined class 'Array' (/angular2_material/lib/src/components/grid_list/grid_list.dart, line 22, col 3)
dist/dart/angular2_material:[warning] Undefined class 'Array' (/angular2_material/lib/src/components/grid_list/grid_list.dart, line 28, col 3)
dist/dart/angular2_material:[warning] The argument type 'num' cannot be assigned to the parameter type 'String' (/angular2_material/lib/src/components/grid_list/grid_list.dart, line 107, col 41)
dist/dart/angular2_material:[warning] The argument type 'num' cannot be assigned to the parameter type 'String' (/angular2_material/lib/src/components/grid_list/grid_list.dart, line 108, col 40)
dist/dart/angular2_material:[hint] Unused import (/angular2_material/lib/src/components/grid_list/grid_list.dart, line 8, col 8)
dist/dart/angular2_material:[hint] The value of the local variable 'percentHeightPerTile' is not used (/angular2_material/lib/src/components/grid_list/grid_list.dart, line 120, col 13)
dist/dart/angular2_material:[hint] Unused import (/angular2_material/lib/src/components/input/input.dart, line 4, col 8)
dist/dart/angular2_material:[hint] Unused import (/angular2_material/lib/src/components/progress-circular/progress_circular.dart, line 4, col 8)
dist/dart/angular2_material:[warning] Undefined class 'Event' (/angular2_material/lib/src/components/radio/radio_button.dart, line 121, col 10)
dist/dart/angular2_material:[warning] Conditions must have a static type of 'bool' (/angular2_material/lib/src/components/radio/radio_button.dart, line 128, col 9)
dist/dart/angular2_material:[warning] Undefined class 'KeyboardEvent' (/angular2_material/lib/src/components/radio/radio_button.dart, line 132, col 13)
dist/dart/angular2_material:[warning] Undefined name 'KeyCodes' (/angular2_material/lib/src/components/radio/radio_button.dart, line 133, col 26)
dist/dart/angular2_material:[warning] Undefined class 'KeyboardEvent' (/angular2_material/lib/src/components/radio/radio_button.dart, line 215, col 13)
dist/dart/angular2_material:[warning] Undefined class 'Array' (/angular2_material/lib/src/components/radio/radio_dispatcher.dart, line 12, col 3)
dist/dart/angular2_material:[hint] Unused import (/angular2_material/lib/src/components/radio/radio_button.dart, line 15, col 8)
dist/dart/angular2_material:[hint] Unused import (/angular2_material/lib/src/components/radio/radio_button.dart, line 21, col 8)
dist/dart/angular2_material:[warning] Undefined class 'Array' (/angular2_material/lib/src/components/radio/radio_dispatcher.dart, line 12, col 3)
dist/dart/angular2_material:[warning] The getter 'disabled_' is not defined for the class 'MdSwitch' (/angular2_material/lib/src/components/switcher/switch.dart, line 41, col 17)
dist/dart/angular2_material:[warning] The setter 'disabled_' is not defined for the class 'MdSwitch' (/angular2_material/lib/src/components/switcher/switch.dart, line 44, col 10)
dist/dart/angular2_material:[warning] The getter 'disabled_' is not defined for the class 'MdSwitch' (/angular2_material/lib/src/components/switcher/switch.dart, line 45, col 34)
dist/dart/angular2_material:[warning] Undefined class 'KeyboardEvent' (/angular2_material/lib/src/components/switcher/switch.dart, line 47, col 13)
dist/dart/angular2_material:[warning] Undefined name 'KeyCodes' (/angular2_material/lib/src/components/switcher/switch.dart, line 48, col 26)
dist/dart/angular2_material:[hint] Unused import (/angular2_material/lib/src/components/switcher/switch.dart, line 6, col 8)
```

**Comment by kevmoo** [comment]
CC @yjbanov

**Comment by yjbanov** [comment]
Fixed by #1411
